api/bazel/repositories.bzl

```diff
@@ -149,3 +154,21 @@ filegroup(
 )
 
 """
+
+ZIPKINAPI_BUILD_CONTENT = """
+load("@envoy_api//bazel:api_build_system.bzl", "api_proto_library")
+load("@io_bazel_rules_go//proto:def.bzl", "go_proto_library")
+api_proto_library(
+    name = "zipkin",
+    srcs = [
+        "zipkin.proto",
+    ],
+    visibility = ["//visibility:public"],
+)
+go_proto_library(
```

Comment:
Please don't have any direct `go_proto_library` in the API tree. Instead use `api_go_proto_library`.

Reply:
Done.